ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -11,42 +11,41 @@
 
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
-import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.FindPatientPage;
+import org.openmrs.reference.page.PatientVisitsDashboardPage;
 import org.openmrs.uitestframework.test.TestData;
 
-import static org.junit.Assert.assertTrue;
-
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 
 public class TransferToWardServiceTest extends ReferenceApplicationTestBase {
 
     private static final String INPATIENT_WARD = "Inpatient Ward";
-
     private static final String ISOLATION_WARD = "Isolation Ward";
-
     private TestData.PatientInfo testPatient;
 
     @Before
     public void createTestData() throws Exception {
         testPatient = createTestPatient();
-        String visitType = TestData.getAVisitType();
-        String visit = new TestData.TestVisit(testPatient.uuid, visitType, getLocationUuid(homePage)).create();
```

Comment:
Refactored the `codeblock ` above so that the test logic is effective. Ideally `transferToWardService` test implementation can be effective when interacting with `PatientVisitDashboard` page **not** `ClinicianFacingPatientDashboard page`. The refactored `codeblock` creates `testvisit.uuid, visistType` and creating `SessionLocation `(`getLocationUuid(homePage`), But creating session location can only be effective when a test class extends `LocationSensitiveApplicationTestBase` so this block `getLocationUuid(homePage)` is not used by the test at all . The other `two methods'` implementation has been catered within the test logic method `LN 40-43` and thus enabling the test to interact with `PatientVisitDashboard` page which holds functionalities that the test implementation needs for it to be effective.